### DISCUSSION TITLE
[QUESTION] Radial gradient with different axis points like Inkscape

### DISCUSSION THREAD

**Discussion by wstaelens** [mono/SkiaSharp#1625] [Q&A]
![skiasharp radial](https://user-images.githubusercontent.com/7447285/107530800-7c65c480-6bbc-11eb-8682-dd0715820c15.PNG)

In SkiaSharp (C#) How do you draw a radial gradient with different axis points like we can do in Inkscape. (see image)?

**Comment by mattleibow** [comment]
You probably want to do a normal radial gradient, but with a local matrix?

**Reply by mattleibow** [reply]
Check this out: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/effects/shaders/circular-gradients

**Reply by mattleibow** [reply]
And maybe this overload: https://docs.microsoft.com/en-us/dotnet/api/skiasharp.skshader.createradialgradient?view=skiasharp-1.68.2#SkiaSharp_SKShader_CreateRadialGradient_SkiaSharp_SKPoint_System_Single_SkiaSharp_SKColor___System_Single___SkiaSharp_SKShaderTileMode_SkiaSharp_SKMatrix_
It has the localMatrix parameter that you can use to control that actual size. Usually it is just a circle, but because you know the center and scale of the handles, you can apply it here. The handles will basically control the matrix.

**Reply by wstaelens** [reply]
ok thanks for the hint to the local matrix!
